client-side/.env

```diff
@@ -1,2 +1,2 @@
 REACT_APP_BASE_URL='http://localhost:3000'
-
+REACT_APP_CLIENT_ID='1079513751617-cvu5tvh0ggnogkvj8im50op7gocnehrj.apps.googleusercontent.com'
```

Comment:
Should this be constant value?

Reply:
yes of corse.
If I won't use it ' it can't work. This is the way to get data from google